<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DecayPathEditor</class>
 <widget class="QWidget" name="DecayPathEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>656</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Beta Feeding Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0">
   <property name="leftMargin">
    <number>11</number>
   </property>
   <property name="topMargin">
    <number>11</number>
   </property>
   <property name="rightMargin">
    <number>11</number>
   </property>
   <property name="bottomMargin">
    <number>11</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabDecay">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="mother">
      <attribute name="title">
       <string>Mother</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="labelMotherIsotope">
         <property name="text">
          <string>Isotope:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelMotherT12">
         <property name="text">
          <string>T1/2 :</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelMotherQvalue">
         <property name="text">
          <string>Q (keV): </string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelMotherSpinParity">
         <property name="text">
          <string>  I:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QTableWidget" name="tableMotherLevels">
         <property name="rowCount">
          <number>6</number>
         </property>
         <property name="columnCount">
          <number>4</number>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>80</number>
         </attribute>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <column/>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Daughter">
      <attribute name="title">
       <string>Daughter</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="labelDaughterIsotope">
         <property name="text">
          <string>Isotope:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="4">
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>141</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Deexcitation model</string>
          </property>
         </widget>
         <widget class="QComboBox" name="comboBoxModel">
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>20</y>
            <width>91</width>
            <height>25</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="buttonApplyModel">
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>50</y>
            <width>89</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </widget>
       </item>
       <item row="11" column="0" colspan="2">
        <widget class="QTableWidget" name="tableDaughterLevels">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="rowCount">
          <number>0</number>
         </property>
         <property name="columnCount">
          <number>4</number>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>80</number>
         </attribute>
         <column/>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QPushButton" name="buttonRemoveLevel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>RemoveLevel</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelDaughterQvalue">
         <property name="text">
          <string>Q (keV):</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QPushButton" name="buttonNormBetaFeeding">
         <property name="toolTip">
          <string>Normalizes beta feding to 100%</string>
         </property>
         <property name="text">
          <string>Norm. beta feeding</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelDaughterT12">
         <property name="text">
          <string>T1/2:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QPushButton" name="buttonAddLevel">
         <property name="toolTip">
          <string>Adds new level</string>
         </property>
         <property name="text">
          <string>Add new level</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1" rowspan="4">
        <widget class="QFrame" name="frame">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="labelPseudoLevels">
            <property name="text">
             <string>Add PseudoLevels</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelPseudoLevelEnergyDist">
            <property name="text">
             <string>Energy dist. (keV)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="linePseudoLevEnStep"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelPseudoLevelEnMin">
            <property name="text">
             <string>Energy min. (keV)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="linePseudoLevEnMin"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelPseudoLevelEnMax">
            <property name="text">
             <string>Energy max. (keV)</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="linePseudoLevEnMax"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="labelPseudoLevelIntensity">
            <property name="text">
             <string>Total Intensity (%)</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="linePseudoLevTotInten"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label">
            <property name="toolTip">
             <string>Gamma intensity model to be used</string>
            </property>
            <property name="text">
             <string>Gamma intensity  model</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="comboBoxIntensity"/>
          </item>
          <item row="6" column="0">
           <widget class="QRadioButton" name="radioStatisticalModelApply">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Apply statistical model for the gamma intensities deexciting pseudo levels</string>
            </property>
            <property name="text">
             <string>apply stat.mod.</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QPushButton" name="buttonAddPseudoLevel">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelTotalIntensity">
         <property name="text">
          <string>Total feeding intensity :</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gradDaughter">
      <attribute name="title">
       <string>GrandDaughter</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="labelGrandDaughterIsotope">
         <property name="text">
          <string>Isotope : </string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelGrandDaughteT12">
         <property name="text">
          <string>T1/2 : </string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelGranDaughterQvalue">
         <property name="text">
          <string>Q : </string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QTableWidget" name="tableGrandDaughterLevels">
         <property name="rowCount">
          <number>0</number>
         </property>
         <property name="columnCount">
          <number>4</number>
         </property>
         <column/>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="buttonClose">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
